doc/Core.xml

```diff
@@ -492,32 +500,34 @@
     <title>Normative references</title>
     <para>IEEE 1003.1, The Open Group Base Specifications Issue 6, IEEE Std 1003.1, 2004 Edition</para>
     <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://pubs.opengroup.org/onlinepubs/009695399/"></link>&gt;</para>
-    <para>IETF RFC 2131, Dynamic Host Configuration Protocol</para>
+    <para>RFC 2131, Dynamic Host Configuration Protocol</para>
     <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.ietf.org/rfc/rfc2131.txt"></link>&gt;</para>
-    <para>IETF RFC 2136, Dynamic Updates in the Domain Name System (DNS UPDATE)</para>
+    <para>RFC 2136, Dynamic Updates in the Domain Name System (DNS UPDATE)</para>
```

Comment:
It looks like ISO/IEC has this format:
INTERNET ENGINEERING TASK FORCE (IETF). RFC 2136

See example in section 10.3 https://www.iso.org/sites/directives/current/part2/index.xhtm